WatchApp Extension/Scenes/GlucoseChartScene.swift

```diff
@@ -64,7 +64,7 @@ struct Scaler {
     func rect(for range: WatchDatedRange) -> CGRect {
         let a = point(range.startDate, range.minValue)
         let b = point(range.endDate, range.maxValue)
-        let size = CGSize(width: b.x - a.x, height: b.y - a.y)
+        let size = CGSize(width: b.x - a.x, height: max(b.y - a.y, 2))
```

Comment:
hardcoding 2 here is probably going to raise some eyebrows. I like the pragmatic aspect of it, but it's going to be an undocumented weirdness in the API.

Perhaps a better approach would be to add a variable?
```
func rect(for range: WatchDatedRange withMinWidth: Int = 2)
```

to make it clearer in the API?

Reply:
Agreed that it's unclear as it stands.  Not clear that the minimum height is worth a variable - maybe just a comment?

Reply:
the advantage of a variable over a comment is that it is fairly obvious to anybody using the API that there's an option. Ie, Xcode will let you know about the behavior. If you want the equivalent, you need to put a comment at the top of the function so that Xcode can display it to the developer. I can see it both ways, but I think a variable is probably the better way to go. You pick, and we'll see what the upstream code reviewers say :-)